### DISCUSSION TITLE
Copilot hints depend on directory name

### DISCUSSION THREAD

**Discussion by Desiders** [community/community#55378] [Copilot Conversations]
### Select Topic Area

Bug

### Body

I came across the fact that Copilot doesn't offer hints if it isn't satisfied with the directory name. If change the directory name and restart VS Code (in my case), then everything starts working.

Directory names:
 - no_bitches_bot (**wrong**)
 - just_test_name (**success**)

[First screenshot](https://ibb.co/cJfLr06) (example with test name)
[Second screenshot](https://ibb.co/4WCYFrg) (example with "bitches" in directory name)

The question arises, why does Copilot filter directory names?

**Comment by bryantson** [comment]
See: https://www.theregister.com/2021/09/02/github_copilot_banned_words_cracked/

**Reply by Desiders** [reply]
> See: https://www.theregister.com/2021/09/02/github_copilot_banned_words_cracked/

```AI programming assistant from responding to input, or generating output, with offensive terms```

```The technical preview includes filters to block offensive words and avoid synthesizing suggestions in sensitive contexts```

How does this relate to the name of the directory when Copilot has nothing to do with it?
